global.R

```diff
@@ -2,38 +2,32 @@
 # Simple Shiny template for annotations utils 
 # search, select, merge and download prototype 
 # ------------------------------------------------------------------------------
-usePackage <- function(p) 
-{
-  if (!is.element(p, installed.packages()[,1]))
-    install.packages(p, dep = TRUE)
-  require(p, character.only = TRUE)
+usePackage <- function(p, github, repos = getOption("repos"), ...) {
+  if (!missing(github)) {
+    ## Install from github -- devtools::install_github will automatically skip
+    ## if SHA of most recent commit matches what's installed already
+    devtools::install_github(paste(github, p, sep = "/"), ...)
+  }
```

Comment:
This doesn't seem to be the case in practice: `usePackage()` re-installs `syndccutils` (and all of its dependencies... from source) every time I call it locally. :( I have `devtools` v1.13.4 installed.

Reply:
What version of git2r do you have installed? I think there was an SSL issue in that package that resulted in `install_github()` always reinstalling. Can you try updating to the current github version of git2r? If that solves it I an update the DESCRIPTION and make sure it gets the right version.

Reply:
I upgraded to v0.21.0 of *git2r*, but that didn't seem to fix the problem. Do you recommend installing the dev version from GitHub (just realized how meta that is...)?

Reply:
Yeah that's what I'd recommend